.github/workflows/pull.yml

```diff
@@ -204,28 +204,14 @@ jobs:
         conda activate "${CONDA_ENV}"
 
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-linux.sh "cmake"
+        
+        # install pybind
+        bash install_requirements.sh --pybind xnnpack
 
         # install Llava requirements
         bash examples/models/llama2/install_requirements.sh
         bash examples/models/llava/install_requirements.sh
 
```

Comment:
why don't we run export_llava.py anymore? looks like test_llava still requires the model to have already been exported?

Reply:
The last test in test_llava.py exports and tests llava